docs/container.md

```diff
@@ -0,0 +1,69 @@
+<!--
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+
+# Containerized imageio server
+
+## Overview
+
+ovirt-imageio provides an image for running the ovirt-imageio-daemon
+in a container.
+
+Using a container, users can run an image transfer pod for every
+transfer, exposing the pod for remote access, and connecting the
+relevant storage the pod. When the transfer is done, the user will
+unexpose the pod and terminate it.
+
+In this mode the imageio server will serve one ticket, injected into the
+pod when starting the pod, instead of adding a ticket using the control
+service. The control service is actually disabled.
+
+To create the container run:
+
+    make container
```

Comment:
This is relevant only for developers. Users should access the container from 
a quay.io - we need to publish the container there.

Lets have a separate section focused on developers (how to build the container, ...)
but keep the main focus on users.